package/MDAnalysis/coordinates/base.py

```diff
@@ -1701,7 +1700,7 @@ def copy(self):
 
         new.ts = self.ts.copy()
         for auxname, auxread in self._auxs.items():
-            new.add_auxiliary(auxname, auxread.copy())
+            new.add_auxiliary(auxread.copy(), auxname)
```

Comment:
We'll eventually want this covered by tests. Not sure why it wasn't originally covered.

Reply:
Ah yes, from what i've seen in the tests there is ` test_copy_with_auxiliary_pyedr()` in `test_copying.py`, which does call the `copy` function from the `ReaderBase` class, but there is no case of testing the `copy` function of the `SingleFrameReaderBase` class